### PR TITLE
Table Branch

### PR DESCRIPTION
**Pull request by LewisWelch94**

### CODE REVIEW

**Comment by OnikNoor97** [comment]
All looks good!
Just one little thing
CONSTRAINT [PK_UserId]		PRIMARY KEY ([UserId])
Primary Key should have a constraint name, please view the example above for User.sql

File: Otal.lmaoo.Database/dbo/Tables/Feature.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿CREATE TABLE [dbo].[Feature]
+(
+  [FeatureId]	 INT	NOT NULL	IDENTITY(1,1),
+  [Name]		 VARCHAR(50)		NOT NULL,
+  [ProjectId]	 INT	NOT NULL,
+  [Active]		 INT	DEFAULT '1',
+
```
Comment (OnikNoor97) [inline Otal.lmaoo.Database/dbo/Tables/Feature.sql:7]: No Primary Key here?

File: Otal.lmaoo.Database/dbo/Tables/Comment.sql [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿CREATE TABLE [dbo].[Comment]
+(
+  [CommentId]		 INT			NOT NULL	IDENTITY(1,1),
+  [CommentContent]	 VARCHAR(255)   NOT NULL,
+  [CommentCreated]   DATETIME		NOT NULL	DEFAULT CURRENT_TIMESTAMP,
+  [TicketId]		 INT			DEFAULT		NULL,
+  [UserId]			 INT			DEFAULT		NULL,
+
+  CONSTRAINT [PK_Comment_CommentId]	PRIMARY KEY ([CommentId]),
+
```
Comment (OnikNoor97) [inline Otal.lmaoo.Database/dbo/Tables/Comment.sql:10]: Remove Extra Space, contraints should be together

File: Otal.lmaoo.Database/dbo/Tables/Ticket.sql [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  [FeatureId]	INT				NOT NULL,
+  [ReporterKey] INT				DEFAULT		NULL,
+  [AssigneeKey] INT				DEFAULT		NULL,
+  [Created]		DATETIME		NOT NULL	DEFAULT CURRENT_TIMESTAMP,
+  [Updated]		DATETIME		NOT NULL,
+  [Progress]	VARCHAR(20)		NOT NULL	DEFAULT 'Open',
+  [Deadline]	DATETIME		NULL DEFAULT NULL,
+
+  CONSTRAINT [PK_Ticket_TicketId]	PRIMARY KEY (TicketId),
+
```
Comment (OnikNoor97) [inline Otal.lmaoo.Database/dbo/Tables/Ticket.sql:14]: contraints should be together

File: Otal.lmaoo.Database/dbo/Tables/ProjectAccess.sql [hunk 4]
```diff
@@ -0,0 +1,11 @@
+﻿CREATE TABLE [dbo].[ProjectAccess] (
+  [ProjectAccessId]		INT			NOT NULL	IDENTITY(1,1),
+  [UserId]				INT			NOT NULL,
+  [ProjectId]			INT			NOT NULL,
+  [ManagerAccess]		BIT			NOT NULL	DEFAULT '0',
+
+  CONSTRAINT [PK_ProjectAccess_ProjectAccessId] PRIMARY KEY ([ProjectAccessId]),
+
```
Comment (OnikNoor97) [inline Otal.lmaoo.Database/dbo/Tables/ProjectAccess.sql:8]: contraints should be together

**Review by OnikNoor97 — CHANGES_REQUESTED** [review]

File: Otal.lmaoo.Database/dbo/Tables/Feature.sql (hunk 1 above)
Comment (LewisWelch94) [reply Otal.lmaoo.Database/dbo/Tables/Feature.sql:7]: Primary key on line 9

**Pull request merged**